pom.xml

```diff
@@ -68,7 +68,13 @@
             <name>Spring Releases</name>
             <url>https://repo.spring.io/libs-release</url>
         </repository>
+        <repository>
+            <id>org.jboss.repository.releases</id>
```

Comment:
Access to JBoss artifacts.
Alot of the dependencies were brought over from the labs.
I've removed the unnecessary ones for this feature including this one.
I've added a comment for each dependency which explains what its for.